### ISSUE TITLE
Need more specific errors

### ISSUE THREAD

**Issue by secretrobotron** [mozilla/petitions.mozilla.org#20]
If your email address is wrong, or if you haven't checked a required checkbox, the error message is the same: "Oops, please complete fields highlighted in red." We should help out a bit more by stating what's wrong.

@lovegushwa @ScottDowne to avoid the clutter of a bunch of simultaneous error messages, does this seem like a good hierarchy?

1. "please fill in required fields" (highlighted in red — just as they are now)
2. "your email address is invalid"
3. "make sure you check the opt in checkbox"

(This is firmly P2.)

**Comment by lovegushwa** [comment]
@secretrobotron @ScottDowne were just talking errors today. Once you get these in the form's let me know and I'll help tweak.

**Comment by ScottDowne** [comment]
@secretrobotron I'm not going to worry about changing the actual error message text, as that'll happen when we paste in the translated text. Can you make sure the errors you want are translated though?

I'll ensure we display the messages in the right cases in the correct places.

**Comment by ScottDowne** [comment]
![petition1](https://cloud.githubusercontent.com/assets/197334/9939049/40873934-5d36-11e5-819b-b889037a20a7.png)

I added an error message for country too, because the generic one seemed really out of place under email, and if I moved it under country, it made it look like it wasn't associated with the name fields.

My copy is a placeholder... Need to get proper error messages translated given the above image. Also the first error message can be specific for first and last name.

https://github.com/mozilla/petitions.mozilla.org/pull/28
